templates/tracker/issue.add.twig

```diff
@@ -87,7 +90,10 @@
             <div class="span9 issue-add-details">
 
                 <div class="alert alert-warning">
-                    {{ "Report security issues to the Joomla! Security Strike Team (JSST) at <a class=\"alert-link\" href=\"mailto:security@joomla.org\">security@joomla.org</a> or with the <a class=\"alert-link\" href=\"https://developer.joomla.org/contact-security-team.html\">JSST contact form</a>, please do not use the public tracker for security issues."|_|raw }}
+                    {{ _(
+                    "Report security issues to the Joomla! Security Strike Team (JSST) at <a %class1% %href1%>security@joomla.org</a> or with the <a %class2% %href2%>JSST contact form</a>, please do not use the public tracker for security issues.",
+                    {"%class1%" : "class=\"alert-link\"", "%href1%" : "href=\"mailto:security@joomla.org\"", "%class2%" : "class=\"alert-link\"", "%href2" : "href=\"https://developer.joomla.org/contact-security-team.html\""}
+                    )|raw }}
                 </div>
```

Comment:
I don't know if this is "good"?
I guess it's "shorter" for translators and slightly more "maintainable"?